wiki/Requeriments_UserStories.md

```diff
@@ -0,0 +1,49 @@
+# Requeriments & User Stories of calculator-angular-tdd
+## Sprint 1
+
+### 1. Funcional requirements 
+
+  1. The user must be able to use a calculator with the functionalities of the operations of multiplying, dividing, adding, subtracting.
+
+  2. The application must support operations between decimals
+  
+  3. Calculator values must be able to be cleaned
+  
+  4. When the value of the calculator is 666 an image of the devil must be displayed
+  
+  5. When the value of the calculator is 42 should be shown to God
+
+### 2. No funcional requirements
+
+  1. The background of the application must have a background of a modern concept (the developer must propose some)
+
+  2. The application must at least look the same or more modern than the example given
+
+### 3. Historias de usuario
+  1. I as a user want to see the value that the calculator currently has, to know the result of my options
+    
+      1. The initial value is "0"
+  
+  2. I as a user must be able to click on buttons to enter number 1-9
+  
+  3. I as a user want to press a button to demarcate the decimal septation of the number
```

Comment:
the decimal separation of the current number

Reply:
Done